src/main/treeDataProviders/dependenciesTree/dependenciesDataProvider.ts

```diff
@@ -102,7 +103,18 @@ export class DependenciesTreeDataProvider implements vscode.TreeDataProvider<Dep
         this._onDidChangeTreeData.fire();
     }
 
-    private async scanAndCacheArtifact(progress: vscode.Progress<{ message?: string; increment?: number }>, checkCanceled: () => void) {
+    private async ScanAndCacheComponents(progress: vscode.Progress<{ message?: string; increment?: number }>, checkCanceled: () => void) {
+        await Promise.all([
+            (async (progress: vscode.Progress<{ message?: string; increment?: number }>, checkCanceled: () => void) => {
+                await this.XrayScanAndCache(progress, checkCanceled);
+            })(progress, checkCanceled),
+            (async (progress: vscode.Progress<{ message?: string; increment?: number }>, checkCanceled: () => void) => {
+                await this.GoCenterScanAndCache(progress, checkCanceled);
+            })(progress, checkCanceled)
+        ]);
+    }
```

Comment:
You don't need 3 progress bars. Please delete the 2 new progress bars.
```suggestion
    private async ScanAndCacheComponents(progress: vscode.Progress<{ message?: string; increment?: number }>, checkCanceled: () => void) {
        await this.xrayScanAndCache(progress, checkCanceled);
        await this.goCenterScanAndCache(progress, checkCanceled);
    }
```